stories/components/menuitems.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import { Menu } from '@headlessui/react';
+import { useMemo } from 'react';
+import { FaMeteor } from 'react-icons/fa';
+
+import { MenuItems, MenuOptions } from '@/components/menu/MenuItems';
+
+export default {
+  title: 'Components / MenuItems',
```

Comment:
You can move these stories and the "Dropdown" stories to a common "Components / DropdownMenu" folder.